CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,6 @@
+Code of Conduct
+===============
+
+The OpenSSL [Code of Conduct] is pulished on the project's website.
```

Comment:
Typo: published

Reply:
Ups, good catch! Typos in the middle of a word are the most dangerous all. The human eye tends to see what it wants to see.

Reply:
I [force-pushed](https://github.com/openssl/openssl/compare/318eee3d3fdbbea853ef7adc36674510fadc0bc7..4b9d2923b31090db732f4b643a62f4a860367579) the correction.